recipes/biobuddy/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "biobuddy" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3e6b0eb3280e872d56945ec986ef7b468f53f4b1b1124c5f899f6fd493e22f30
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
+    - pip
+    - setuptools
+    - hatchling
+    - wheel
+  run:
+    - python >=3.10
+    - numpy
+    - ezc3d
+    - lxml
+    - scipy
+test:
+  imports:
+    - biobuddy
+  requires:
+    - numpy
+    - ezc3d
+    - lxml
+    - scipy
+    - biorbd
+  commands:
+    - python -c "import biobuddy"
+
+about:
+  home: https://github.com/pyomeca/biobuddy
+  summary: 'A generic interface to generate a virtual buddy'
+  description: |
+    A generic interface to generate a virtual buddy
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
```

```suggestion
  license_file: LICENSE
```
https://github.com/pyomeca/biobuddy/blob/main/LICENSE